p2p/simulations/session_controller.go

```diff
@@ -84,6 +105,66 @@ func NewSessionController() (*ResourceController, chan bool) {
 						parent.SetResource(conf.Id, m)
 					}
 					parent.id++
+					
```

Comment:
I see now this should probably be sourced out to a separate "plugin" folder or something. That would also demonstrate how people write their own handles for debug, node etc.